RecoBTag/Combined/python/combinedMVAV2Computer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+combinedMVAV2Computer = cms.ESProducer("CombinedMVAV2JetTagESProducer",
+	jetTagComputers = cms.vstring(
+		'jetProbabilityComputer',
+		'jetBProbabilityComputer',
+		'combinedSecondaryVertexV2Computer',
+		'softPFMuonComputer',
+		'softPFElectronComputer'
+	),
+	weightFile = cms.FileInPath('RecoBTag/Combined/data/CombinedMVAV2_13_07_2015.weights.xml.gz')
```

Comment:
Here I would suggest to introduce switches similar to those recently introduced for soft lepton tagger. In summary, GBRForest would be enabled by default but the possibility to use plain TMVA would still be there. In addition, everything would also be set up for a future switch to the CondDB.